arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -367,7 +367,7 @@
     },
     "/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/servers": {
       "get": {
-        "description": "List all the Analysis Services servers for the given subscription",
+        "description": "List all the Analysis Services servers for the given subscription.",
```

Comment:
"Gets" but then "List" -- might be good to standardize on one form or the other, but perhaps not in this PR.